install/index.md

```diff
@@ -163,38 +163,42 @@ The same instructions as [for Linux](#nixos_nix_nixpkgs) apply when running on m
 
 # LCG releases on CVMFS
 
-## Pre-built ROOT without dependencies
+## Standalone ROOT
 
-If your platform mounts [CVMFS](https://cernvm.cern.ch/portal/filesystem){:target="\_blank"} (as, for example, CERN LXPLUS does),
-ROOT is directly available as an [LCG release](http://lcginfo.cern.ch/){:target="\_blank"}.
+If your platform mounts [CVMFS](https://cernvm.cern.ch/portal/filesystem){:target="\_blank"}
+ROOT is directly available via [LCG releases](http://lcginfo.cern.ch/){:target="\_blank"}.
 
-ROOT installations with minimal external dependencies are available for Fedora, Ubuntu, CentOS 7 and MacOS at:
+ROOT installations with minimal external dependencies are available for Fedora, Ubuntu, CentOS7 and MacOS at:
 
 ```
 /cvmfs/sft.cern.ch/lcg/app/releases/ROOT/<version>/<platform>
 ```
 
-For example, to set up ROOT 6.22/00 on a CentOS7 machine that already has all [ROOT required dependencies]({{'/install/dependencies' | relative_url}}) installed, just run:
+For example, to set up ROOT 6.24/00 on a CentOS7 machine, just run:
 
 ```
-source /cvmfs/sft.cern.ch/lcg/app/releases/ROOT/6.22.00/x86_64-centos7-gcc48-opt/bin/thisroot.sh
+source /cvmfs/sft.cern.ch/lcg/app/releases/ROOT/6.24.00/x86_64-centos7-gcc48-opt/bin/thisroot.sh
 ```
 
-## ROOT plus dependencies
+Make sure to use the same compiler in use by the release in your working environment.
```

Comment:
I don't understand what this warning is trying to warn about. Maybe "Make sure you use the distribution's default compiler, just like this ROOT build"?

Reply:
yes that's what i wanted to warn about, will rephrase

Reply:
Just merge when done, please